po/es.po

```diff
@@ -772,9 +773,9 @@ msgid ""
 "a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class="
 "\"gnucash\">GnuCash</span> wikis."
 msgstr ""
-"Otros recursos son los wikis de <span class=\"gnucash\">GnuCash</span> en <a "
-"href=\"https://wiki.gnucash.org/wiki/\">inglés</a> o <a href=\"https://wiki."
-"gnucash.org/wiki/Es/\">español</a>."
+"Otros recursos son los wikis <a href=\"https://wiki.gnucash.org/wiki/\""
+">inglés</a> o <a href=\"https://wiki.gnucash.org/wiki/De/\">alemán</a> de "
+"<span class=\"gnucash\">GnuCash</span>."
```

Comment:
What is wrong with our spanish wiki page?
Did you not see the comment `Translators: If wiki pages in your laguage exists, consider to replace "De"/"Deutsch" by your language.`?